src/plugins/cosmos/components/modals/Send/views/Details.tsx

```diff
@@ -0,0 +1,259 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
```

Comment:
This module is a duplication of the existing `<Details />` with the addition of these JSX nodes:
https://github.com/shapeshift/web/pull/1238/files#diff-32a7b208eee8c7e78e369610d1f12492c5538b5dbaa8ff9f7c0540de4ad2a6e5R185-R232

The reason for the duplication is that it consumes the Cosmos `SendInput` which is different for the existing one, and trying to use both `ensName` and `memo` on the existing `<Confirm />` would make us fight with types and is also blending too much chain-specific stuff in a general-purpose component, while chain-specific logic should live in plugins.